python/cugraph-nx/cugraph_nx/tests/test_convert.py

```diff
@@ -0,0 +1,202 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import cupy as cp
+import networkx as nx
+import pytest
+
+import cugraph_nx as cnx
+from cugraph_nx import interface
+
+
+@pytest.mark.parametrize("graph_class", [nx.Graph, nx.DiGraph])
+@pytest.mark.parametrize(
+    "kwargs",
+    [
+        {},
+        {"preserve_edge_attrs": True},
+        {"preserve_node_attrs": True},
+        {"preserve_all_attrs": True},
+        {"edge_attrs": {"x": 0}},
+        {"edge_attrs": {"x": None}},
+        {"edge_attrs": {"x": cnx.convert.REQUIRED}},
+        {"edge_attrs": {"x": ...}},  # sugar for REQUIRED
+        {"edge_attrs": "x"},
+        {"node_attrs": {"x": 0}},
+        {"node_attrs": {"x": None}},
+        {"node_attrs": {"x": cnx.convert.REQUIRED}},
+        {"node_attrs": {"x": ...}},  # sugar for REQUIRED
+        {"node_attrs": "x"},
+    ],
+)
+def test_convert_empty(graph_class, kwargs):
+    G = graph_class()
+    cG = cnx.from_networkx(G, **kwargs)
+    H = cnx.to_networkx(cG)
+    assert G.number_of_nodes() == cG.number_of_nodes() == H.number_of_nodes() == 0
+    assert G.number_of_edges() == cG.number_of_edges() == H.number_of_edges() == 0
+    assert cG.edge_values == cG.edge_masks == cG.node_values == cG.node_masks == {}
+    assert G.graph == cG.graph == H.graph == {}
+
+
+def test_convert():
```

Comment:
Wow, this test is testing a lot.  Would this be better as separate tests more focused on specific aspects of converting? OTOH, with `pytest --pdb` it may not matter much in practice, so I'm not too concerned about diagnosing a failing test that's this big.

Reply:
Heh, yeah, I know. It's probably better to break it up, but then we need to figure out how to name all the tests ;)

Reply:
ok yeah, coverage is the priority and it looks good from that perspective, so maybe a FIXME would suffice.

Reply:
Adding FIXME comment.

At a glance, breaking this up looks kind of annoying, because we sometimes mutate the current graph to change its characteristics. And you're exactly right: this was done to increase coverage by testing more cases. Having unit tests be, well, a single unit is nice to have, but having tests is better than no tests, so I'm fine with adding the FIXME for now.